site/docs/v3-adoption-progress.md

```diff
@@ -0,0 +1,173 @@
+| **Avro** | ✅ **Full Support** | Complete with `VariantConversion` and `VariantLogicalType` | Full read/write support |
+| **Parquet** | ✅ **Full Support** | Complete with `VariantWriterBuilder` and `VariantReaderBuilder` | Supports both serialized and shredded variants |
+| **ORC** | ❌ **Not Implemented** | Schema conversion exists but throws `UnsupportedOperationException` | No actual read/write implementation |
+
+### Geometry/Geography Type File Format Support
+
+| **File Format** | **Core Support** | **Implementation Status** | **Notes** |
+|---|---|---|---|
+| **Avro** | ❌ **Not Implemented** | Type definitions exist but no data processing | No conversion or logical type support |
+| **Parquet** | ❌ **Not Implemented** | Type definitions exist but no data processing | No read/write implementation |
+| **ORC** | ❌ **Not Implemented** | Type definitions exist but no data processing | No read/write implementation |
+
+### Timestamp Nanoseconds Support
+
+| **Engine** | **timestamp_ns** (nanos) | **timestamptz_ns** (nanos) | **Notes** |
+|---|---|---|---|
+| **Core** | ✅ **Full Support** | ✅ **Full Support** | Complete support for all timestamp types and precisions |
+| **Spark** | ❌ **Not Implemented** | ❌ **Not Implemented** | Only supports microsecond precision timestamps |
+| **Flink** | ✅ **Full Support** | ✅ **Full Support** | Complete support for all timestamp types and precisions |
+
+### Unknown Type Support
+
+| **Engine** | **Core Support** | **Implementation Status** | **Notes** |
+|---|---|---|---|
+| **Core** | ✅ **Full Support** | Complete type system and schema evolution support | Supports optional unknown fields, proper validation |
+| **Spark v4.0+** | ✅ **Full Support** | Complete type mapping to `NullType` | Full support with comprehensive tests |
+| **Spark v3.4/v3.5** | ❌ **Not Implemented** | Missing from type conversion | Throws "Cannot convert unknown type to Spark" error |
+| **Flink** | ✅ **Full Support** | Complete type mapping to `NullType` | Full support across all versions (v1.20, v2.0, v2.1) |
+
+### Multi-Argument Transforms Support
+
+| **Engine** | **Z-Order Support** | **Implementation Status** | **Notes** |
+|---|---|---|---|
+| **Core** | ✅ **Full Support** | Complete `Zorder` class implementation | Full support for multi-column transforms in partitioning and sorting |
+| **Spark** | ✅ **Full Support** | Complete `SparkZOrderFileRewriteRunner` implementation | Full support across all versions (v3.4, v3.5, v4.0) with file rewriting |
+| **Flink** | ❌ **Not Implemented** | No multi-argument transform support | Only single-argument transforms supported |
+
+### Default Values Support
+
+| **Engine** | **initial-default** | **write-default** | **Notes** |
+|---|---|---|---|
+| **Core** | ✅ **Full Support** | ✅ **Full Support** | Complete support for both types with schema evolution |
+| **Spark** | ✅ **Full Support** | ✅ **Full Support** | Full support across all versions (v3.4, v3.5, v4.0) |
```

Comment:
"Full read/write support ..." but no DDL support at the moment.

Reply:
Thank you. Updated the table